mock/mock-query.js

```diff
@@ -1,9 +1,15 @@
 const daily = require('./daily-counts');
 const totals = require('./dataset-totals');
 const groups = require('./group-data');
+const dailySearches = require('./daily-total-searches');
+const allTime = require('./all-time-searches');
 
 module.exports = {
   usageByDateType: () => Promise.resolve(daily),
   usageByType: () => Promise.resolve(totals),
-  usageByGroup: () => Promise.resolve(groups)
+  usageByGroup: () => Promise.resolve(groups),
+  searchTotals: () => Promise.resolve(allTime),
+  // eslint-disable-next-line no-dupe-keys
+  searchTotals: () => Promise.resolve(dailySearches),
```

Comment:
What's the meaning of this? Looks like the previous line just gets overridden.

Reply:
this should have been fixed